xsgen/brightlite.py

```diff
@@ -65,3 +70,14 @@ def write(self, libs, dirname):
             f.write("\n".join(cladrows))
             f.write("\n")
         shutil.copyfile("TAPE9.INP", os.path.join(dirname, "TAPE9.INP"))
+        # write cross section json file
+        xsdata = [[[nucname.name(int(n)), rxname.name(int(r)), float(x)] 
```

Comment:
Maybe just turn these into a couple of for loops? Nested list comprehensions can be a little hard to read sometimes. Non-blocking.